NEWS

```diff
@@ -6,6 +6,58 @@ releases.  For more information about the current release, please
 consult RELEASE-NOTES.  For more information about changes, please
 consult ChangeLog.
 
+Invenio v1.0.10 -- released 2016-11-09
+--------------------------------------
+
+New features
+~~~~~~~~~~~~
+
++ installation
+
+  - Initial release of kickstart installations scripts. Ported from
+    the old `tiborsimko/invenio-devscripts` repository of helper
+    scripts and adapted to resemble Invenio 3 kickstart installation
+    scripts. Tested on Ubuntu 12.04 and CentOS 6.
+
+Improved features
+~~~~~~~~~~~~~~~~~
+
++ I18N
+
+  - Updates Catalan, French, German, Italian, Russian, Slovak Spanish
```

Comment:
I don't remember if the release notes are auto generated or not for `main-1.x` but, I guess there is an "and" missing between Slovak and Spanish